src/util.ts

```diff
@@ -317,6 +317,30 @@ export function removeQuotes(str: string): string {
     return str;
 }
 
+// Remove only he quotes that are surrounding the given string.
+export function removeSurroundingQuotes(str: string): string {
+    str = str.trim();
+    if (str.startsWith('"') && str.endsWith('"')) {
+        str = str.substring(1, str.length - 1);
```

Comment:
>str.length - 1 [](start = 31, length = 14)

Can you double check that this is supposed to be an index instead of a length?

Reply:
Yes, index.